content/ios/query.md

```diff
@@ -98,6 +98,27 @@ SKYDatabase *privateDB = database;
 }];
 ```
 
+You can query by fields on a referenced record. Following the above example, if
+we want to narrow orders placed in italian restaurants only:
+
+```obj-c
+NSPredicate *predicate = [NSPredicate predicateWithFormat:@"resturant.cuisine = %@", @"italian"];
```

Comment:
s/resturant/restaurant

There are 5 occurrences under the whole "Reference" section, can you change the others too?